extensions/amp-audio/0.1/amp-audio.js

```diff
@@ -108,15 +119,53 @@ export class AmpAudio extends AMP.BaseElement {
   pauseCallback() {
     if (this.audio_) {
       this.audio_.pause();
+      if (getMode().test) {
+        this.isPlaying_ = false;
+      }
+    }
+  }
+
+  pause() {
+    const storyEl = closestByTag(this.element, 'AMP-STORY');
+    if (this.audio_ && !storyEl) {
+      this.audio_.pause();
+      if (getMode().test) {
+        this.isPlaying_ = false;
+      }
     }
   }
 
+  play() {
+    const storyEl = closestByTag(this.element, 'AMP-STORY');
+    if (this.audio_ && !storyEl) {
+      this.audio_.play();
+      if (getMode().test) {
+        this.isPlaying_ = true;
+      }
+    }
+  }
+
+  /**
+   * Returns whether the audio is playing or not.
+   * @returns {boolean}
+   * @VisibleForTesting
+   */
+  playerStatus() {
```

Comment:
This won't be removed from the final binary. I suggest you just check for `isPlaying_`.

Reply:
Done in PR here # 15222